app/models/search.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class Search
+  include ActiveModel::Model
+
+  attr_accessor :query, :planning_application_ids
```

Comment:
Passing the planning application ids to the search because they've already been filtered by things like local authority, user type and view type.